oap-application/oap-application/src/main/java/oap/application/Kernel.java

```diff
@@ -88,6 +88,7 @@ public class Kernel implements Closeable, AutoCloseable {
     final String name;
     private final List<URL> moduleConfigurations;
     private final Supervisor supervisor = new Supervisor();
+    private ApplicationConfiguration applicationConfiguration = new ApplicationConfiguration();
```

Comment:
could you please explain more in detail each of 
```
* APPLICATION_STOP_DETECT_TIMEOUT - default 5s
* APPLICATION_FORCE_ASYNC_AFTER_TIMEOUT - default false
```
Maybe explain the whole flow and how each of the these will be used.